pkgs/misc/screensavers/physlock/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  version = "git-20150126";
+  name = "physlock-${version}";
+  src = fetchFromGitHub {
```

Comment:
I think it's preferrable to `fetchurl` the [stable release](https://github.com/muennich/physlock/releases/tag/v0.4.5) here